openwhisk/_test/first/3/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Do we have a chance to remove binary and zip from git and generate it on the flight when the test is run? (Did not have a chance to comment under them, that's why here)

Reply:
those are required for the test - I need a BINARY in elf format to check if the binary is recognized...